imap_processing/cdf/config/imap_ultra_global_cdf_attrs.yaml

```diff
@@ -0,0 +1,151 @@
+Descriptor: &desc
+  Ultra>IMAP Ultra-Energy (IMAP-Ultra) Energetic Neutral Atom Imager
+TEXT: &txt >
+  Ultra captures images of very energetic neutral atoms, particularly hydrogen (H) atoms,
+  produced in the solar system at the heliosheath, the region where the solar wind slows,
+  compresses, and becomes much hotter as it meets the interstellar medium (ISM).
+  The instrument also measures the distribution of solar wind electrons and protons, and
+  the magnetic field. See https://imap.princeton.edu/instruments/ultra for more details.
+
+imap_ultra_l1b_45sensor-de:
+    Descriptor: 45sensor-de
```

Comment:
I think this descriptor should be something like `ULTRA45> IMAP Ultra-Energy Energetic Neutral Atom Imager` or something like that?[ ISTP standard](https://spdf.gsfc.nasa.gov/istp_guide/attributes.html#Descriptor)